iota/transaction/base.py

```diff
@@ -570,11 +570,23 @@ def as_tryte_string(self):
             + self.nonce
         )
 
-    def get_signature_validation_trytes(self):
+    def get_bundle_essence_trytes(self):
```

Comment:
💭 This is probably low-level enough that it won't break any application logic, but maybe we should make this part of the next minor release (e.g., v2.3.0 => v2.4.0), just to be safe; what do you think?

Reply:
Probably no application uses it, unless it chooses to create the bundle hash itself.
It will be part of the next release, I'm just wondering what is the right approach here. We have in `develop`:
- the async changes, which is pretty big with dropping Python 2  and 3.5 support, and adding the asynchronous api.
- the new networking lib `httpx` with connection pooling is quite faster than `requests`.
- renaming `testnet` to `devnet`, this is a breaking change for apps that relied on it.

Should we include all these in a next minor release 2.4.0, or we say we move to PyOTA 3.0.0 since we dropped py 2 support anyway? :smiley_cat:
Or we wait until summer and reserve PyOTA 3.0.0 for the chrysalis updates?
Probably there will also be some more extended api calls and maybe even account module coming before chrysalis though...

Reply:
Ooh, right.  🤔 Dropping Python 2 support is definitely big enough for a major release.  Makes sense to me to bundle it all up in PyOTA 3.

There's no rule that says we have to wait for a long time between major releases; I figure there's no need to make devs wait until Chrysalis, even if that ends up being big enough to justify PyOTA 4 😸